apps/meteor/client/views/admin/subscription/components/cards/PlanCard/PlanCardCommunity.tsx

```diff
@@ -12,12 +12,12 @@ const PlanCardCommunity = (): ReactElement => {
 		<Card height='full'>
 			<PlanCardHeader name={t('Community')} />
 			<CardBody flexDirection='column'>
-				<CardRow>
+				<Box display='flex' flexDirection='row' alignItems='center'>
```

Comment:
i think we should stay with CardRow dont you think? otherwise we are going to only use box for everything, and bye bye componentization (look just repeated yourself) twice here

Reply:
In this specific situation, I believe there is no problem with the boxes as they are part of arbitrary custom content, like the seats and MACs counter cards of the Subscription page. Also, CardRow was not made to be used with vertical Cards, and using it here caused issues with the CardBody content